packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -21,24 +21,16 @@ let ReleaseGoldMultiSigProxy: ReleaseGoldMultiSigProxyContract
 let ReleaseGold: ReleaseGoldContract
 let ReleaseGoldProxy: ReleaseGoldProxyContract
 const ONE_CGLD = web3.utils.toWei('1', 'ether')
+const MAINNET_START_TIME = new Date('22 April 2020 16:00:00 UTC').getTime() / 1000
 
 async function handleGrant(releaseGoldConfig: any, currGrant: number) {
   console.info('Processing grant number ' + currGrant)
 
-  // Special mainnet string is intended as MAINNET+X where X is months after mainnet launch.
-  // This is to account for the dynamic start date for mainnet,
-  // and some grants rely on x months post mainnet launch.
-  let releaseStartTime: any
-  if (releaseGoldConfig.releaseStartTime.startsWith('MAINNET')) {
-    const addedMonths = Number(releaseGoldConfig.releaseStartTime.split('+')[1])
-    const date = new Date()
-    if (addedMonths > 0) {
-      date.setDate(date.getDate() + addedMonths * 30)
-    }
-    releaseStartTime = date.getTime() / 1000
-  } else {
-    releaseStartTime = new Date(releaseGoldConfig.releaseStartTime).getTime() / 1000
-  }
+  // Sentinel MAINNET dictates a start time of mainnet launch, April 22 2020 16:00 UTC in this case
+  const releaseStartTime = releaseGoldConfig.releaseStartTime.startsWith('MAINNET')
```

Comment:
What happened to `addedMonths`?

Reply:
Yeah grants that were originally MAINNET+2 were changed to June 19 2020 to make this more explicit